src/components/organisms/AnimateMapOnboardFlow/AnimateMapOnboardModal.tsx

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import { Modal } from "react-bootstrap";
+import classNames from "classnames";
+
+import { ButtonNG } from "components/atoms/ButtonNG";
+
+import { AnimateMapOnboardFlowProps } from "./AnimateMapOnboardFlowProps";
+
+import "./AnimateMapOnboardModal.scss";
+
+export interface AnimateMapOnboardModalContentProps {
+  className?: string;
+  descriptionBottom?: string;
+  descriptionTop?: string;
+  disabledNext?: boolean;
+  nextText?: string;
+  posterAlt?: string;
+  posterSrc?: string;
+  show?: boolean;
+  skipText?: string;
+  title?: string;
+}
+
+export type AnimateMapOnboardModalProps = AnimateMapOnboardFlowProps &
+  AnimateMapOnboardModalContentProps;
+
+export const AnimateMapOnboardModal: React.FC<AnimateMapOnboardModalProps> = ({
```

Comment:
A modal dialog that serves as a step container for onboarding flow. It is made with preset props for quick bootstrap of a flow as well as embedded content.